pkgs/os-specific/bsd/freebsd/pkgs/boot-install.nix

```diff
@@ -1,10 +1,20 @@
 { buildPackages, freebsd-lib }:
 
-# Wrap NetBSD's install
+# Wrap GNU coreutils' install
 buildPackages.writeShellScriptBin "boot-install" (
   freebsd-lib.install-wrapper
   + ''
+    fixed_args=()
+    while [[ ''${#args[0]} > 0 ]]; do
+      case "''${args[0]}" in
+        -l)
```

Comment:
Would be nice to explain in comment or commit message what -l does and why it's safe to ignore.

Reply:
Done

Reply:
Fixed!